pkg/apis/noobaa/v1alpha1/noobaa_types.go

```diff
@@ -365,6 +365,10 @@ type NooBaaStatus struct {
 	// BeforeUpgradeDbImage is the db image used before last db upgrade
 	// +optional
 	BeforeUpgradeDbImage *string `json:"beforeUpgradeDbImage,omitempty"`
+
+	// PostgresMajorVersion is the major version of postgress db image
+	// +optional
+	PostgresMajorVersion int `json:"postgresMajorVersion,omitempty"`
```

Comment:
Use a string with const values like we do with `UpgradePhase`. it will give us more flexibility and is more readable

Reply:
sure